pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java

```diff
@@ -118,9 +119,14 @@ private MethodHandles.Lookup getLookup(Class<?> anchorClass) throws IllegalAcces
     @Override
     public void ensureClassInitialized(Class<?> cl)  {
         try {
+            /**/
+            Class.forName(cl.getName(), true, cl.getClassLoader());
+            /*/
```

Comment:
I would also remove the commented out code.

Reply:
> I would also remove the commented out code and empty comments.

Yes, those were part of my experiment for how to get this to work. I have now cleaned them up.